### PR TITLE
DockerFix

### PR DESCRIPTION
**Pull request by sim2github**
My solution to fix #413 and #439.

### CODE REVIEW

**Comment by sim2github** [comment]
Anyone have worked nginx config for single domain installation? My solution don't work propertly.

File: docker-compose.yml [hunk 1]
```diff
@@ -1,59 +1,54 @@
 data:
+  container_name: 'data'
```
Comment (trntv) [inline docker-compose.yml]: We are using `docker-compose` it is a bad idea to set predefined container name. If you are working on several projects for example.

File: docs/installation.md [hunk 2]
```diff
@@ -252,27 +252,20 @@ upstream php-fpm {
...
-`docker-compose run cli help`
-
-`docker-compose run cli migrate`
-
-`docker-compose run cli rbac-migrate`
+`docker exec -it fpm console/yii help`
 
 2. How to connect to the application database with my workbench, navicat etc?
-MySQL is available on `127.0.0.1`, port `33060`. User - `root`, password - `root`
+MySQL is available on `192.168.99.100`, port `3306`. User - `root`, password - ``
```
Comment (trntv) [inline docs/installation.md]: It is boot2docker ip address. Native docker implementations for OSX and Windfow has been released.

**Comment by trntv** [comment]
Thank you for your work, but i'm starting to work on docker changes myself to fix all this issues at a time

File: docker-compose.yml (hunk 1 above)
Comment (sim2github) [inline docker-compose.yml]: I agree its unnecessary. Add usage in docs.

File: docs/installation.md (hunk 2 above)
Comment (sim2github) [inline docs/installation.md]: Its beta-release (works only on 10.10+ and win10). So DockerToolbox steel in game. Add variation in docs.

**Comment by sim2github** [comment]
I'm done version that satisfy me but still not perfect to release. So i'm ready to discuss.

**Pull request closed without merging**